features/preferences/impl/src/main/kotlin/io/element/android/features/preferences/impl/root/PreferencesRootState.kt

```diff
@@ -33,6 +33,7 @@ data class PreferencesRootState(
     val showDeveloperSettings: Boolean,
     val showLockScreenSettings: Boolean,
     val showNotificationSettings: Boolean,
+    val showBlockedUsersItem: Boolean,
```

Comment:
I guess something is missing, since I do not see the usage of `showBlockedUsersItem` in `PreferencesRootView`.

Reply:
I think my push should have been a force push, so it was rejected 🤦 .